src/Views/Wallpaper.vala

```diff
@@ -148,20 +148,18 @@ public class PantheonShell.Wallpaper : Gtk.Box {
         var filter = new Gtk.FileFilter ();
         filter.add_mime_type ("image/*");
 
-        var chooser = new Gtk.FileChooserNative (
-            _("Import Photo"), null, Gtk.FileChooserAction.OPEN,
-            _("Import"),
-            _("Cancel")
-        );
-        chooser.filter = filter;
-        chooser.select_multiple = true;
-
-        chooser.show ();
-        chooser.response.connect ((response) => {
-            if (response == Gtk.ResponseType.ACCEPT) {
-                var files = chooser.get_files ();
-                for (var i = 0; i <= files.get_n_items (); i++) {
-                    var file = (File) files.get_item (i);
+        var file_dialog = new Gtk.FileDialog () {
+            accept_label = _("Import"),
+            default_filter = filter,
+            modal = true,
+            title = _("Import Photo")
```

Comment:
Should we use this to imply that you can import multiple wallpapers at once? Something like "Import Photo(s)" or "Wallpaper(s)" if we want to change the wording entirely. Maybe this will be too difficult for translations though.

Reply:
I'm down with that. Probably just changing it to "Import Photos" is good?